CONTRIBUTING.md

```diff
@@ -117,6 +117,80 @@ You can learn more about [pull requests on GitHub](https://help.github.com/en/gi
 
 You can also contribute to the project without specifically writing content for the site. Many discussions are happening all around the project. Protocol Labs, and by extension, IPFS, is a collaborative company that relies heavily on the community. We need your opinions and input! If you have an idea about a topic or want to weigh in on a discussion, feel free to have your say either on [GitHub](https://github.com/ipfs/ipfs-docs/issues) or on the [IPFS forums](https://discuss.ipfs.tech/).
 
+## Create a new page
+
+Sometimes you will need to make a whole new page for content to live. The process to do this is outlined below.
+
+These steps were performed in Visual Studio Code (VSCode).
+
+The first step to creating a new page is to identify the type of content it will be to determine where it will live in the structure of the docs. It may need to be a whole new section or it could be added to an existing section as a sub-page. An important concept to keep in mind when deciding where to put the new content is the flow of the already existing content. You wouldn't want to put a page or section designed for experienced users between the already existing **Install** and **Basics** sections.
```

Comment:
I like that you're going into all the things that the contributor should think about before adding a page. You could rephrase it as a series of questions. Something like:
```
Before creating a new page, ask yourself the following questions:

- What type of content am I creating? A tutorial, a conceptual explanation, or a few grammatical edits?
- Who is my intended audience? New developers, seasoned Web3 veterans, or the general community?
- Does this content require a whole new page or can it be condensed into a section or sub-page?
- Can an existing page have the new content added to it, instead of creating a whole new page?
- If a new page must be added, how will it fit into the overall flow and structure of the existing documentation?

Thinking about these questions before creating a new page benefits you and the PLN community. For example, putting a page or section designed for experienced users between the current **Install** and **Basics** sections might negatively impact the documentation experience for the community.
```

Reply:
Agree, I think this works better. Implemented in the newest version.